### PR TITLE
Add Aryan-Srivastava

### PR DESCRIPTION
**Pull request by Aryan-Srivastava**

### CODE REVIEW

File: public/data/Aryan.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Aryan Srivastava",
+  "bio": "Android and open source enthusiast",
+  "avatar": "https://github.com/Aryan-Srivastava.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "http://github.com/FayasNoushad",
```
Comment (rohan-kulkarni-25) [inline public/data/Aryan.json:8]: - Please Update your github Url

**Review by rohan-kulkarni-25 — CHANGES_REQUESTED** [review]

File: public/data/Aryan.json (hunk 1 above)
Comment (FayasNoushad) [reply public/data/Aryan.json:8]: Change that link to `https://github.com/Aryan-Srivastava`.

Comment (schmelto) [inline public/data/Aryan.json:8]:
```suggestion
      "url": "https://github.com/Aryan-Srivastava",
```

**Review by schmelto — CHANGES_REQUESTED** [review]

**Review by schmelto — APPROVED** [review]
lgtm

File: public/data/Aryan.json (hunk 1 above)
Comment (Aryan-Srivastava) [reply public/data/Aryan.json]: Thanks for telling.

Comment (Aryan-Srivastava) [reply public/data/Aryan.json]: Thanks for pointing it out.

**Pull request merged**